test/cpp/util/grpc_cli.cc

```diff
@@ -86,6 +90,7 @@ DEFINE_string(output_binary_file, "",
 DEFINE_string(metadata, "",
               "Metadata to send to server, in the form of key1:val1:key2:val2");
 DEFINE_string(proto_path, ".", "Path to look for the proto file.");
+DEFINE_string(proto_file, "", "Name of the proto file.");
```

Comment:
Let's call it protofiles. I think we need to support a list rather than just a file (which can be done in a later pr).